suggest Strimzi only for development (#920)
lightbend/cloudflow: docs/shared-content-source/docs/modules/administration/pages/how-to-install-and-use-strimzi.adoc

include::ROOT:partial$include.adoc[]

> NOTE: This setup is meant only for development purposes! For production make sure to choose a suitable Kafka solution from an upstream provider.

Reviewer: ```suggestion
NOTE: This setup is meant only for development purposes! For production make sure to choose a suitable Kafka service.
```
